less/v1/items.less

```diff
@@ -438,3 +445,74 @@
     padding: 0.1em 0.5em;
   }
 }
+
+/* ----------------------------------------- */
+/*  Summoning Configuration                  */
+/* ----------------------------------------- */
+
+.dnd5e.summoning-config {
+  .unbutton {
+    width: unset;
+    border: none;
+    background: none;
+    line-height: unset;
+
+    &:hover, &:focus { box-shadow: none; }
+    &:hover { text-shadow: 0 0 8px var(--color-shadow-primary); }
+    &:focus-visible { outline: 2px solid black; }
+  }
+
+  .form-header {
+    justify-content: space-between;
+    button { flex: unset; }
+  }
+
+  ul.profiles {
+    padding: 0;
+    list-style: none;
+    gap: 12px;
+  }
+  li.profile {
```

Comment:
I wanted something that made the distinctions between the groups read a bit clearer than the V1 styles, especially when more fields are added. I figured since I was going to be designing something distinct anyway, I might as well lean into V2 styling rather than come up with something now that would just have to be redesigned later, but I understand there is a bit of a difference between the list and the rest of the form.

Reply:
Here's what the Equipment Advancement config looks like, which also uses this style:

<img width="456" alt="Screenshot 2024-02-28 at 15 43 56" src="https://github.com/foundryvtt/dnd5e/assets/19979839/48731e61-0525-490b-aad5-45d92f89560a">

Reply:
I think it looks a little out-of-place personally, but I'm not fussed, so long as it works. We will have to re-evaluate all the styling for these form-based dialogs once we have the item sheet styles nailed down anyway.